src/Microsoft.AspNet.Razor.Runtime/TagHelperContentWrapperTextWriter.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text;
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    /// <summary>
+    /// Implements a <see cref="TextWriter"/> that wraps around <see cref="TagHelperContent"/>.
+    /// </summary>
+    public class TagHelperContentWrapperTextWriter : TextWriter
```

Comment:
So here's the problem: We have to copy from a `StringCollectionTextWriter` to `TagHelperContent`. `StringCollectionTextWriter` does not expose the `Buffer`. Moreover it is present in `Mvc.Core`. So it does not know about `TagHelperContent` and we cannot refer `Razor` in `Mvc.Core`. Hence this approach.